src/pudl/workspace/datastore.py

```diff
@@ -442,13 +442,15 @@ def parse_command_line():
     parser.add_argument(
         "--gcs-cache-path",
         type=str,
-        help="Load datastore resources from Google Cloud Storage. Should be gs://bucket[/path_prefix]",
+        help="""Load datastore resources from Google Cloud Storage. Should be gs://bucket[/path_prefix].
+                If specified without --bypass-local-cache, the local cache will be populated from the GCS cache.
+                If specified with --bypass-local-cache, the GCS cache will be populated by Zenodo.""",
```

Comment:
I think it will take this white space literally and add 16 spaces to the beginnings of the 2nd and third lines in the help message won't it?

Maybe we shouldn't be so vague about the URL and just say it should probably be `gs://zenodo-cache.catalyst.coop`

Reply:
I fixed the formatting and added `gs://zenodo-cache.catalyst.coop` to the help message.